foundations/foundation-vertx/src/main/java/io/servicecomb/foundation/vertx/client/tcp/TcpResponseCallback.java

```diff
@@ -18,6 +18,6 @@
 
 import io.servicecomb.foundation.vertx.AsyncResultCallback;
 
-public interface TcpResonseCallback extends AsyncResultCallback<TcpData> {
+public interface TcpResponseCallback extends AsyncResultCallback<TcpData> {
```

Comment:
We need to still keep the old wrong API with deprecated annotation to warn the user.

Reply:
fixed, thanks for pointing out.